api_spec.yaml

```diff
@@ -1231,11 +1231,217 @@ paths:
+                                                              type: boolean
+                                                          providers:
+                                                              type: array
+                                                              items:
+                                                                  $ref: "#/components/schemas/thirdPartyProvider"
+                                                  firstFactors:
+                                                      type: array
+                                                      items:
+                                                          type: string
+                                                      example:
+                                                          - "emailpassword"
+                                                          - "thirdparty"
+                                                  requiredSecondaryFactors:
+                                                      type: array
+                                                      items:
+                                                          type: string
+                                                      example:
+                                                          - "otp-email"
+                                                  coreConfig:
+                                                      type: object
+                                                  userCount:
+                                                      type: number
+                                                  validFirstFactors:
+                                                      type: array
+                                                      items:
+                                                          type: string
+                                                      example:
+                                                          - "emailpassword"
+                                                          - "thirdparty"
+                                                      description: The first factors that the tenant can use based on the recipes initialized and login methods enabled
+                                                  mergedProvidersFromCoreAndStatic:
+                                                      type: array
+                                                      items:
+                                                          $ref: "#/components/schemas/thirdPartyProvider"
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "UNKNOWN_TENANT_ERROR"
+                401:
+                    description: Unauthorised access
+                    content:
```

Comment:
How are you handling tenant not found?

Reply:
In the node SDK when tenant is not returned from the API, we return this status

Reply:
https://github.com/supertokens/supertokens-node/blob/8b1636886dedda42ea835e49937257b93942abd9/lib/ts/recipe/dashboard/api/multitenancy/getTenantInfo.ts#L68-L76